ush/nems_configure.sh

```diff
@@ -56,7 +56,7 @@ if [[ "${cplflx}" = ".true." ]]; then
   local CPLMODE="${cplmode}"
   local coupling_interval_fast_sec="${CPL_FAST}"
   local RESTART_N="${restart_interval}"
-  local ATMTILESIZE="${CASE:2:}"
+  local ATMTILESIZE="${CASE:1}"
```

Comment:
Is there an issue open for this bug that should be added to the PR?

Reply:
No issue.  It was identified in the PR #1865 opened by @rmontuoro 
Since that PR will not be merged, I just pulled the fix out from that PR.
Same with the bug in `parsing_namelists_FV3.sh`